### PR TITLE
review.ninja testing

### PR DESCRIPTION
**Pull request by AvalancheYT**
Test

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/ChickenDevs/ChickenFreedomMod/pull/6" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ You may download official binaries from the [releases page](https://github.com/T
 
 ### Contributing ###
 Please see [CONTRIBUTING.md](CONTRIBUTING.md) if you are interested in developing TotalFreedomMod. For information on how TotalFreedomMod is licensed, please see [LICENSE.md](LICENSE.md).
+
```
Comment (AvalancheYT) [inline README.md]: A line note, test for ```review.ninja```

Comment (AvalancheYT) [inline README.md]: !fix

Comment (AvalancheYT) [inline README.md]: !fixed

Comment (AvalancheYT) [inline README.md]: !fixed

**Pull request closed without merging**